### PR TITLE
New Adapter: InfyTV

### PR DESCRIPTION
**Pull request by infytvcode**

### CODE REVIEW

**Comment by mansinahar** [comment]
@infytvcode Looks like there's a test failure due to URL mismatch in one of your test cases. Can you please fix this when you have a chance?

File: adapters/infytv/infytv.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type InfytvAdapter struct {
```
Comment (hhhjort) [inline adapters/infytv/infytv.go:15]: You can call this simply "Adapter", the "Infytv" identification is already supplied by the package name. As you have it, referencing your adapter from outside the package would be `infytv.InfytvAdapter` which looks a little redundant.

Comment (infytvcode) [reply adapters/infytv/infytv.go:15]: @hhhjort Thanks for the suggestion. Will update it.

**Comment by infytvcode** [comment]
Documentation Pull request https://github.com/prebid/prebid.github.io/pull/3865

File: adapters/infytv/params_test.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	}
+
+	for _, p := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderInfyTV, json.RawMessage(p)); err == nil {
+			t.Errorf("Schema allowed invalid params: %s", p)
+		}
+	}
+}
+
+var validParams = []string{
```
Comment (hhhjort) [inline adapters/infytv/params_test.go:36]: You may want to include placementId in some of your tests.

Comment (infytvcode) [reply adapters/infytv/params_test.go:36]: @hhhjort I have added placementId in the test. 
Thanks

**Review by hhhjort — DISMISSED** [review]

File: adapters/infytv/infytv.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type Adapter struct {
```
Comment (SyntaxNode) [inline adapters/infytv/infytv.go:15]: Actually, we would prefer the name `adapter`. The beginning upper case letter tells Go to export this type and there is no need for that. Sorry to contradict with @hhhjort here.

File: static/bidder-info/infytv.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+maintainer:
+  email: "tech+hb@infy.tv"
+capabilities:
```
Comment (SyntaxNode) [inline static/bidder-info/infytv.yaml:3]: I would like to confirm InfyTV does not have an IAB Europe GVL vendor id.

File: adapters/infytv/infytv.go (hunk 3 above)
Comment (infytvcode) [reply adapters/infytv/infytv.go]: @SyntaxNode Updated to `adapter`.

File: static/bidder-info/infytv.yaml (hunk 4 above)
Comment (infytvcode) [reply static/bidder-info/infytv.yaml:3]: @SyntaxNode Yes, InfyTV does not have an IAB Europe GVL vendor id.

**Review by hhhjort — APPROVED** [review]

**Review by SyntaxNode — APPROVED** [review]

**Pull request merged**